src/app/tab2/tab2.page.html

```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
-</ion-content>
-
-<ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
-  <ion-img [src]="photo.webviewPath" (click)="showActionSheet(photo, position)"></ion-img>
-</ion-col>
-
-<ion-content>
-  <ion-grid>
-    <ion-row>
-      <ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
-        <ion-img [src]="photo.webviewPath"></ion-img>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
-
   <ion-content>
+    <ion-grid>
+      <ion-row>
+        <ion-col
+          size="6"
```

Comment:
I'm a little confused at what happened here? Is there a best practice that says they must be on different lines, also why the space at the end?

Reply:
Its mostly a rule of how easy is to read and to edit, I use a beautifier tool to do that automatically, usually it is better to use different lines. That being said the rules is really just based on how long a line is and if the line would over flow most screens.

Reply:
Oh, I get what you're saying, my screen is basically a TV in my room, so it doesn't usually overflow, I shall keep that in mind.